Formula/votca.rb

```diff
@@ -0,0 +1,40 @@
+class Votca < Formula
+  desc "Versatile Object-oriented Toolkit for Coarse-graining Applications"
+  homepage "https://www.votca.org/"
+  url "https://github.com/votca/votca/archive/refs/tags/v2023-rc.1.tar.gz"
+  sha256 "b992d94b92dd501a32f2378609577aa94f4af06e529431859a082b5390add137"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "eigen"
+  depends_on "fftw"
+  depends_on "gcc" # for OpenMP
+  # add gromacs back once it was built with clang
+  depends_on "hdf5"
+  depends_on "python@3.11"
+
+  uses_from_macos "expat"
+
+  on_macos do
+    depends_on "libomp"
+  end
+
+  def install
+    args = [
+      "-DINSTALL_RC_FILES=OFF",
+      "-DINSTALL_CSGAPPS=ON",
+      "-DBUILD_XTP=OFF",
+      "-DCMAKE_DISABLE_FIND_PACKAGE_GROMACS=ON",
+      "-DENABLE_RPATH_INJECT=ON",
+    ]
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, *args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    system "#{bin}/csg_property", "--help"
```

Comment:
Can we do something more clever here? Run a small analysis (maybe a trivial xyz file?)

Reply:
I added running `ctest` above that will run the whole test suite.

Reply:
No, please remove `ctest` and add a small functional test in the `test do` block. See for example `lammps`

Reply:
Ok, I added another test here.